data/functions/individual.sql

```diff
@@ -34,3 +45,28 @@ begin
   );
 end
 $$ language plpgsql immutable;
+
+
+-- unitemized contributions should not be included in the state breakdowns
+create or replace function is_unitemized(memo_text text) returns bool as $$
```

Comment:
It looks like this was included in the other PR and is now resulting in merge conflicts here.